engine/common/components/src/main/scala/pl/touk/nussknacker/engine/common/components/DecisionTable.scala

```diff
@@ -21,6 +21,8 @@ object DecisionTable extends EagerService with SingleInputDynamicComponent[Servi
 
   override type State = Unit
 
+  private type DecisionTableResponse = java.util.List[java.util.Map[String, Any]]
```

Comment:
maybe just `Output`?

Reply:
Good point, it will be more generic - changed